### PR TITLE
Fix: Social Share - Pinterest Share not working because missing image issue.

### PR DESCRIPTION
**Pull request by amithire1201**
### Description

https://trello.com/c/UwgCz2d4/88-social-share-pinterest-share-not-working-because-missing-image

### Types of changes

Breaking change

### How has this been tested?

- Create Social share in master.
- Switch to this branch, add a featured image to the page check Pinterest share working.
- Create Reusable block use on a different pages and check its working.

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Review by vrundakansara — COMMENTED** [review]
Please solve the below-listed issues.
Also, test this with Reusable blocks functionality.

File: classes/class-uagb-block-js.php [hunk 1]
```diff
@@ -134,7 +134,12 @@ public static function get_social_share_js( $attr, $id ) {
 						if( social_url == "mailto:?body=" ) {
 							target = "_self";
 						}
-						var request_url = social_url + window.location.href;
+						var  request_url ="";
+						if( social_url.indexOf("/pin/create/link/?url=") !== -1) {
```
[the remark this answers is not in the document]
Comment (AbhijitNage123) [reply classes/class-uagb-block-js.php:137]: Does not get the type of social share here.

**Review by vrundakansara — APPROVED** [review]
Let's get this tested once.

**Review by vrundakansara — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -164,6 +164,7 @@ When you use the Ultimate Addons for Gutenberg along with the free Astra theme,
 * Improvement: Inline Notice - New layout added for inline notice.
 * Improvement: Twenty Seventeen Theme : Added compatibility to Twenty Seventeen Sections.
 * Fix: FAQ Schema - Toggle issue when we add multiple FAQ Blocks on same page.
+* Fix: Social Share - Pinterest Share not working because missing image issue.
```
Comment (vrundakansara) [inline README.md:167]: `Fix: Social Share - Resolved the missing image issue in the Pinterest social platform.`

**Pull request merged**